### PR TITLE
Basic finsight transfer by address

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: zingocli/tests/integration_tests.rs [hunk 1]
```diff
@@ -2295,4 +2308,66 @@ async fn zero_value_change() {
...
+            None,
+        )])
+        .await
+        .unwrap();
+
+    utils::increase_height_and_sync_client(&regtest_manager, &recipient, 5)
+        .await
+        .unwrap();
+
+    println!("{}", recipient.do_list_transactions().await.pretty(4));
```
Comment (zancas) [inline zingocli/tests/integration_tests.rs:2332]: I guess we can change these to assert_eq!

File: zingocli/tests/integration_tests.rs [hunk 2]
```diff
@@ -2295,4 +2308,66 @@ async fn zero_value_change() {
...
+        .unwrap();
+    utils::increase_height_and_sync_client(&regtest_manager, &recipient, 5)
+        .await
+        .unwrap();
+
+    println!("{}", recipient.do_list_transactions().await.pretty(4));
+    println!("{}", recipient.do_balance().await.pretty(4));
+    println!(
+        "{}",
+        JsonValue::from(recipient.do_list_txsummaries().await).pretty(4)
```
Comment (zancas) [inline zingocli/tests/integration_tests.rs:2368]: Also...   a climactic assertion would be a nice thing to have in a test.  ;)

File: zingolib/src/wallet/transactions.rs [hunk 3]
```diff
@@ -48,6 +45,12 @@ impl TransactionMetadataSet {
         21
     }
 
+    pub fn get_fee_by_txid(&self, txid: &TxId) -> u64 {
```
Comment (zancas) [inline zingolib/src/wallet/transactions.rs:48]: Nice.

**Review by AloeareV — COMMENTED** [review]
Great! I've written the last half-dozen commits here, so I rule that another reviewer (@zancas is sufficient) is needed for merge.

**Review by AloeareV — APPROVED** [review]

**Pull request merged**